library-extensions-expandable/src/main/java/com/mikepenz/fastadapter/expandable/ExpandableExtension.kt

```diff
@@ -267,15 +286,15 @@ class ExpandableExtension<Item : GenericItem>(private val fastAdapter: FastAdapt
      * @param position the global position of the current item
      * @return a set with the global positions of all expanded items on the root level
      */
-    fun getExpandedItemsRootLevel(position: Int): IntArray {
-        val expandedItemsList = ArraySet<Int>()
+    fun getExpandedItemsRootLevel(position: Int): List<Int> {
```

Comment:
Breaking API change?

Reply:
Yes. That's another reason to go for v4.1.x 

I think the usage of this should be == 0, and migration should be super simple. (considered a new function and deprecate the old, but not sure if it is worth it)